css/style.css

```diff
@@ -0,0 +1,903 @@
+  *,
+*::after,
+*::before {
+    box-sizing: border-box;
+}
+
+html {
+    scroll-behavior: smooth;
+    /* свойство scroll-behavior не наследуется, применяется к прокручиваемым блокам */
+}
+
+ul {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+    color: inherit;
+    display: inline-block;
+}
+
+.header {
+    font-family: Montserrat, sans-serif;
+    height: 806px;
+    background-color: #eff0f2;
+}
+
+.container {
+    width: 1140px;
+    margin: 0 auto;
+}
+
+.header_logo {
+    margin-left: 0%;
+    padding-top: 5.7%;
+}
+
+.header_text {
+    margin-top: 16.7%;
+    margin-left: 5%;
+    text-align: center;
+    margin-bottom: 10%;
+}
+
+.header_text__style {
+    margin-top: 200px;

```

Comment:
How do you choice to use % or px for the same properties?

Reply:
Thanks . I fixed it
```suggestion
  .header_text__style {
      margin-top: 200px;
      margin-bottom: 137px;
      margin-top: 17%;
      margin-bottom: 12%;
      text-align: center;
      font-size: 45px;
      line-height: 59px;
@@ -119,7 +119,7 @@
  .main-information {
      display: flex;
      justify-content: space-around;
      padding-top: 141px;
      padding-top: 9%;
  }
```